bw2io/importers/ecospold1.py

```diff
@@ -82,18 +93,71 @@ def __init__(
             )
         )
 
-
 class NoIntegerCodesEcospold1Importer(SingleOutputEcospold1Importer):
+    """
+    An importer class that deletes integer codes from ecospold1 datasets.
+    
+    Parameters
+    ----------
+    SingleOutputEcospold1Importer : class
+        The base importer class.
+    
+    Attributes
+    ----------
+    strategies : list
+        A list of strategies that the importer applies to process the dataset.
+    
+    Returns
+    -------
+    None
+    """
     def __init__(self, *args, **kwargs):
+        """
+        Initialize NoIntegerCodesEcospold1Importer.
+        
+        Parameters
+        ----------
+        *args : tuple
+            Variable length argument list.
+        **kwargs : dict
+            Arbitrary keyword arguments.
+        
+        Returns
+        -------
+        None
+        """
         super(NoIntegerCodesEcospold1Importer, self).__init__(*args, **kwargs)
         self.strategies.insert(0, delete_integer_codes)
 
 
 class MultiOutputEcospold1Importer(SingleOutputEcospold1Importer):
-    """Import and process mutli-output datasets in the ecospold 1 format.
-
-    Works the same as the single-output importer, but first allocates multioutput datasets."""
```

Comment:
@ShanikaWickramasinghe, where in the updated docstring is this information?

Reply:
I have missed that line. Updated the PR with a new commit https://github.com/brightway-lca/brightway2-io/pull/163/commits/a963d95fe7ec8325d05a6d6a7a80b0e2805c77ca